src/Simulation.cpp

```diff
@@ -1204,10 +1227,9 @@ void Simulation::simulateOneTimestep()
 	} else if ( _temperatureControl != nullptr) {
 		// mheinen 2015-07-27 --> TEMPERATURE_CONTROL
 		_temperatureControl->DoLoopsOverMolecules(_domainDecomposition, _moleculeContainer, _simstep);
-	}
-	// <-- TEMPERATURE_CONTROL
-
-
+	} else if (_temperatureObserver != nullptr) {
+        _temperatureObserver->step(_moleculeContainer);
+    }
```

Comment:
- It was initially created due to my misunderstanding of the formula. I thought we needed the current temperature in a specific region. However, that is not the case.
- But since we still require a method of disabling all other thermostat functionality, it seemed viable to keep the Temperature Observer as a "thermostat" that does nothing. The other option would be using an existing thermostat and setting certain values to zero, which seemed to me to be prone to misconfiguration.